data/transactions/logic/README.md

```diff
@@ -221,6 +221,9 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | `gloads i` | push Ith scratch space index of the Xth transaction in the current group |
 | `gaid t` | push the ID of the asset or application created in the Tth transaction of the current group |
 | `gaids` | push the ID of the asset or application created in the Xth transaction of the current group |
+| `txnas f` | pop an index A. push Ath value of the array field F of the current transaction |
+| `gtxnas t f` | pop an index A. push Ath value of the array field F from the Tth transaction in the current group |
+| `gtxnsas f` | pop an index A and an index B. push Bth value of the array field F from the Ath transaction in the current group |
```

Comment:
The convention, when an opcode has only one stack argument, is to call it `X`.  A bit weord, but let's stick to it until/unless we change it everywhere.

Reply:
Sounds good - I'll keep the `gtxnsas` description, since it pops two stack arguments then.